BrainPortal/app/controllers/signups_controller.rb

```diff
@@ -166,7 +166,16 @@ def index #:nodoc:
 
     scope_default_order(@scope, 'created_at', :desc)
 
-    @base_scope       = Signup
+    if params[:view_hidden].present?
```

Comment:
This entire block of added lines is clumsy. Why not simply:

```ruby
   view_hidden = params[:view_hidden].presence == "true"
   @scope.custom[:view_hidden] = view_hidden
   @base_scope = Signup.where(:hidden => view_hidden)
```

THis doesn the same thing, in three lines instead of nine, without an `if..end` block inside a `if...else...end`